test/docfx.Test/build/TocTest.cs

```diff
@@ -13,7 +13,7 @@ public static class TocTest
             new Report(),
             Directory.GetCurrentDirectory(),
             "en-us",
-            JsonUtility.Deserialize<Config>("{'output': { 'json': true } }".Replace('\'', '\"')),
+            JsonUtility.Deserialize<Config>("{'output': { 'json': true } }".Replace('\'', '\"'), null),
```

Comment:
consider input `string.Empty` instead of `null` here? we can have a non-null check for file